### ISSUE TITLE
Hints dont show up on page scroll

### ISSUE THREAD

**Issue by paradox460** [1995eaton/chromium-vim#357]
When I scroll down on the page, and hit f, I don't get any new hint markers.

They previously worked fine.

Chrome 46.0.2490.13 beta (64-bit), OS X 10.10.5

**Comment by 1995eaton** [comment]
On every page? Also, are you using the GitHub version of cVim (which commit if so) or the Chrome Web Store version?

**Comment by paradox460** [comment]
Every page I've seen. They show up "above the fold," but when I scroll down, they disappear.

I am running cVim 1.2.79 from the Chrome Web Store.

https://gist.github.com/paradox460/e08b1828f0a1ab8b0098 is my cVim config gist

**Comment by 1995eaton** [comment]
Hints are only created in the viewing window (the current links visible on-screen). Are you activating link hints and then scrolling down without closing the hints?

**Comment by paradox460** [comment]
No, I load up a new page, scroll down, hit `f`, and no link hints appear. If I scroll back up to the top of the page, the link hints are on the first *n* links, but no further, no matter what position in the page I am at

**Comment by paradox460** [comment]
Interestingly enough, the hints have started working again. I didn't change any of my settings, but maybe it was a chrome update that fixed it.

Thanks!

**Comment by sivan** [comment]
Same problem with the Chrome Web Store version  in Chrome Stable 46.0.2490.71 (64-bit), OS X 10.10.5.

Hint marks only shows in the first screen. Also, hit `gg` doesn't scroll to top now(or `G` to the bottom). Seems like a bug with page scroll.

**Comment by paradox460** [comment]
Interestingly enough, I started getting issues with cVim again about a week and a half ago.

Chrome Stable 47.0.2526.106 and beta 48.0.2564.48 both display the issues.

Same issue as the original ticket, but with the addition that page movement commands (h,j,k,l,d,u) do not work.

I found disabling the [enable-experimental-web-platform-features](chrome://flags/#enable-experimental-web-platform-features) flag fixed everything.

**Comment by sivan** [comment]
Thanks @paradox460 , also fix my problem annoyed me almost three months.
